### PR TITLE
Animation Storyboard

### LINKED ISSUE #55: [DOCS] - Design Report Animation: Storyboard
Create an outline of the various "scenes" of the Design Report Animation.

"_You are required to submit a 5-minute (maximum length) design animation demonstrating a simulation of your food production technology under operation. This animation should include key elements, such as: the setup, any operation from a user perspective, various inputs and outputs and the shutdown and cleaning process. It is recommended that the animation be saved on a platform that support video files (e.g., YouTube, Google Drive) and provide a link of the video in your submission._"

- Simulation under operation
  - Setup (assembly, startup)
  - Operation (autonomous, from user perspective (maintenance, #76, cleaning))
  - Inputs (Planting) and Outputs (Harvest, Seed Collection)
  - Shutdown and Stowage

### TODO:
- [x] Figure out missing/blank frames
  - [x] A3
  - [x] B1f
  - [x] B1g
  - [x] B1h
  - [x] C1
    - [x] #77 
  - [x] D2
  - [x] D3
  - [x] D4
    - [x] #81 
  - [x] D5?
    - [x] #82 
  - [x] E1
- [x] Figure out if there are any other maintenance procedures
- [x] Frames for each maintenance procedure (B2a-B2h)
- [x] Figure out all the CADs we need, open an issue (under #28)

### Notes

B1k - More abstract animation of a machine learning model? Showing data flow and "learning"

### PR DESCRIPTION
**Pull request by JLefebvre55**
Closes #55

### CODE REVIEW

**Comment by nchareunsouk** [comment]
@JLefebvre55 @narvcharv 
**The frame revisions folder contains all the new frames and is only temporary to make reviewing the new frames easier.
FOLDER SHOULD BE REMOVED BEFORE MERGING**

**Comment by JLefebvre55** [comment]
@nchareunsouk Ok everything looks fantastic. I think we should flesh out startup and shutdown procedures - can you see if this is covered in the design report somewhere?

**Comment by nchareunsouk** [comment]
@JLefebvre55 2.3.6 Crew Time Requirement - Setup & Maintenance does not cover a startup and shut down procedure.

**Review by nchareunsouk — DISMISSED** [review]
👍

**Comment by JLefebvre55** [comment]
> @JLefebvre55 2.3.6 Crew Time Requirement - Setup & Maintenance does not cover a startup and shut down procedure.

@nchareunsouk I think it should (it _is_ "Crew Time Requirement - **Setup** and Maintenance" after all) - I'll open a new issue for adding that to the design report once we have the procedures fleshed out.

**Comment by JLefebvre55** [comment]
@nchareunsouk Everything looks fantastic. Some final requests before merge:
- [x] Merge the new folder with the old one (overwrite old frames)
- [x] Brainstorm more ideas for B1k (optimization)
  - Neural network animation?
  - Gradient descent animation?
  - Plant function digitization (sum of inputs -> sum of outputs, "surrogate model", "scanning" a plant -> computer)

**Review by nchareunsouk — DISMISSED** [review]
👍

**Comment by JLefebvre55** [comment]
@nchareunsouk 
- [ ] A2a to A2h
- [ ] B3
- [ ] Check README list of timings (and make sure sum < 300)
- [ ] Change frame durations to match

**Review by nchareunsouk — APPROVED** [review]
no need for storyboard - following script instead
